Geometry/CaloTopology/src/HcalTopology.cc

```diff
@@ -188,6 +199,28 @@ bool HcalTopology::validDetId(HcalSubdetector subdet, int ieta, int iphi,
   return validHcal(id);
 }
 
+bool HcalTopology::validHT(const HcalTrigTowerDetId& id) const {
+
+  if (id.iphi()<1 || id.iphi()>72 || id.ieta()==0) return false;
```

Comment:
@davidlange6 AT the moment there is no way I can change the hardwired #'s. Please integrate it at the earliest because the HcalDetId changes are waiting for this PR to be integrated.
________________________________
From: David Lange [notifications@github.com]
Sent: 10 November 2015 20:42
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx47 Add trigger primitive version in geometry (#12301)


In Geometry/CaloTopology/src/HcalTopology.cc<https://github.com/cms-sw/cmssw/pull/12301#discussion_r44454520>:

> @@ -188,6 +199,28 @@ bool HcalTopology::validDetId(HcalSubdetector subdet, int ieta, int iphi,
>    return validHcal(id);
>  }
>
> +bool HcalTopology::validHT(const HcalTrigTowerDetId& id) const {
> +
> +  if (id.iphi()<1 || id.iphi()>72 || id.ieta()==0) return false;


@bsunanda<https://github.com/bsunanda> - there is no alternative to these hardwired numbers?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/12301/files#r44454520>.